exchanges/huobi/huobi_wrapper.go

```diff
@@ -705,6 +705,28 @@ func (h *HUOBI) UpdateAccountInfo(ctx context.Context, assetType asset.Item) (ac
 		}
 
 	case asset.CoinMarginedFutures:
+		// fetch swap account info
+		acctInfo, err := h.GetSwapAccountInfo(ctx, currency.Pair{})
+		if err != nil {
+			return info, err
+		}
+
+		var mainAcctBalances []account.Balance
+		for x := range acctInfo.Data {
+			mainAcctBalances = append(mainAcctBalances, account.Balance{
+				CurrencyName: currency.NewCode(acctInfo.Data[x].Symbol),
+				TotalValue:   acctInfo.Data[x].MarginBalance,
+				Hold:         acctInfo.Data[x].MarginFrozen,
+			})
+		}
+
+		info.Accounts = append(info.Accounts, account.SubAccount{
```

Comment:
Just a note, we are not explicitly loading sub account ID `sub_uid` this differentiation I assume can be done in a separate PR when needed. Let me know your thoughts on this. I will add a public trello item.

Reply:
we'll be drilling further into futures shortly, i'm also taking a note for tracking